src/reducers/columnState.js

```diff
@@ -28,23 +29,35 @@ const DEFAULT_STATE = {
   useGroupHeader: false,
 };
 
-function columnStateReducer(state = DEFAULT_STATE, action) {
+function columnState(state = DEFAULT_STATE, action) {
   switch (action.type) {
-    case ActionTypes.INITIALIZE:
-      let { props, columnData, useGroupHeader } = action;
-      return columnStateHelper.initialize(state, props, columnData, useGroupHeader);
-    case ActionTypes.PROP_CHANGE:
-      const { newProps, columnData, useGroupHeader } = action;
+    case ActionTypes.INITIALIZE: {
+      const { props } = action;
+      const {
+        columnGroups,
+        elementTemplates,
+        useGroupHeader,
+      } = convertColumnElementsToData(props);
+
+      return columnStateHelper.initialize(state, props, columnGroups, useGroupHeader, elementTemplates);
```

Comment:
We don't want the elementTemplates here because they're not serializable.  I think you'll have issues using redux devtools if you do this.  Can we just keep the element templates in the React component state?